File: hw/basurto/u3/hw18Farm/JavaScript/OOPConcepts/models/Cut.js
class Cut {

    constructor(id, description, procedure, weight) {
        this.id = id;
        this.description = description;
        this.procedure = procedure;
        this.weight = weight;

    }

    toDocument() {

        return {
            _id: this.id,
            description: this.description,
            procedure: this.procedure,
            weight: this.weight
        };

    }

}

module.exports = Cut;

File: hw/tipan/u3/hw18OOPFarm/OPPConceptsZooJava/src/main/java/ec/edu/espe/oppconcetszoo/model/Cut.java
package ec.edu.espe.oppconcetszoo.model;

public class Cut {

    private int id;
    private String description;
    private String procedure;
    private float weight;

    public Cut(int id, String description, String procedure, float weight) {
        this.id = id;
        this.description = description;
        this.procedure = procedure;
        this.weight = weight;
    }

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getDescription() {
        return description;
    }

    public void setDescription(String description) {
        this.description = description;
    }

    public String getProcedure() {
        return procedure;
    }

    public void setProcedure(String procedure) {
        this.procedure = procedure;
    }

    public float getWeight() {
        return weight;
    }

    public void setWeight(float weight) {
        this.weight = weight;
    }

}